test/dash/dash_parser_manifest_unit.js

```diff
@@ -1828,6 +1828,7 @@ describe('DashParser Manifest', () => {
   });
 
   it('Does not error when image adaptation sets are present', async () => {
+    const idUri = 'http://dashif.org/guidelines/thumbnail_tile';
```

Comment:
I would suggest giving this a more meaningful name like `thumbnailScheme`, and perhaps moving it to a higher scope (like the next highest `describe` block) so it can be shared among tests.